CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 3.3.2 (Under development)
 
+### App Center
+
+ * **[Breaking change]** Bump the minimum Android SDK version to 21 API level.
```

Comment:
Please add more details here (security issue because of cert etc). Also, add mapping to android version, not just API level

```suggestion
* **[Breaking change]** Bump the minimum Android SDK version to 21 API level.
```